filters/map_composition.py

```diff
@@ -79,82 +86,139 @@ def responseComplete(self):
             else:
                 remove_qml = False
 
-            if exists(project_path):
-                if not overwrite:
-                    msg = 'PROJECT is already existing : %s \n' % project_path
-                    request.appendBody(msg)
-                    return
-                else:
-                    remove(project_path)
+            if exists(project_path) and overwrite:
+                # Overwrite means create from scratch again
+                remove(project_path)
+
+            sources_parameters = params.get('SOURCES')
```

Comment:
I changed the params name because now it's not just files.
But we still support old name